diff --git a/node_modules/native-base/index.d.ts b/node_modules/native-base/index.d.ts
index dd661c0..4fea323 100644
--- a/node_modules/native-base/index.d.ts
+++ b/node_modules/native-base/index.d.ts
@@ -887,7 +887,7 @@ declare module "native-base" {
 			contentStyle?: RnViewStyleProp;
 			renderHeader?: (item: any, expanded: boolean, index: number) => React.ReactElement<any>;
 			renderContent?: (item: any, index: number) => React.ReactElement<any>;
-			expanded?: number;
+			expanded?: number[];
 			icon?: string;
 			expandedIcon?: string;
 			iconStyle?: RnTextStyleProp;
